README.md

```diff
@@ -482,9 +660,9 @@ The sample-app/ folder in this repository contains a very simple integration of
 4. Confirm your FNS number on the Forage dashboard ([sandbox](https://dashboard.sandbox.joinforage.app/login/) | [prod](https://dashboard.joinforage.app/login/))
 5. Place your bearer token and FNS number in constants inside sample-app/java/com.joinforage.android.example/ui/complete.flow/tokens/model/TokensUIDefaultState.kt
    1. The sample-app will prompt you for a bearer token and FNS number on the first page of the app, but takes defaults from this file location
-6. Choose the appropriate build variant (usually sandboxDebug for your first run)
+6. Choose the `prod` build variant since all builds [infer `prod` or `sandbox` from the auth token](#a-note-on-build-variants)
 7. Run the sample-app on your emulated device of choice
-8. Use any 16 digit card number to complete the payment flow
+8. Use any 16 to 19 digit card number to complete the payment flow
```

Comment:
Can link or reference [this document](https://docs.joinforage.app/docs/test-ebt-cards#valid-ebt-test-card-numbers). The 9999-prefix passes validation checks and is a standard to reduce confusion

```suggestion
8. Use any 16 to 19 digit card number starting with "9999" to complete the payment flow
```

Reply:
love this - and thanks for the link!